client/src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+
+ReactDOM.render(
+  <div>Hello World</div>,
+  document.getElementById('app')
+);
```

Comment:
nit: can you add a blank line in the end of files to avoid the "no new line at end of file" github alert?

Reply:
Thanks Nanda! Just resolved all listed issues and pushed.